diesel/src/migrations/migration.rs

```diff
@@ -0,0 +1,166 @@
+use connection::Connection;
+use super::{MigrationError, RunMigrationsError};
+
+use std::path::{Path, PathBuf};
+
+pub trait Migration {
+    fn version(&self) -> i64;
```

Comment:
Could this be a string? Feels like the property you want here is sortable, not "only a number". For example: leading zeroes might be useful, or users might want to have hexadecimal or similar. Thoughts?